views/reducers/index.js

```diff
@@ -1,10 +1,10 @@
-import { combineReducers } from 'redux';
-import Camera from './Camera';
+import {combineReducers} from 'redux';
 import Browser from './Browser';
+import CharacterCamera from './CharacterCamera';
 
 const rootReducer = combineReducers({
-  Camera,
-  Browser,
+    CharacterCamera,
+    Browser,
```

Comment:
They are called by the composed reducer, they don't need to listen to any action for the moment.